libqtile/notify.py

```diff
@@ -125,7 +125,10 @@ def add(self, notif):
             self.notifications.append(notif)
             notif.id = len(self.notifications)
             for callback in self.callbacks:
-                callback(notif)
+                try:
+                    callback(notif)
+                except Exception:
```

Comment:
Odd to me that this doesn't trigger some sort of linter error (we have noqa E722 for this elsewhere it looks like), but overall the series looks good.

Reply:
I thought the same when I wrote it and read up that it's only bare `except:`that gives the error, and that `except Exception:` suppresses it because it's explicit that you want all to be caught.